.mergify.yml

```diff
@@ -39,12 +56,7 @@ pull_request_rules:
       - "status-success=ci/centos/upgrade-tests-rbd"
       - "status-success=DCO"
     actions:
-      merge:
-        method: rebase
-        rebase_fallback: merge
-        bot_account: ceph-csi-bot
-        strict: smart
-        strict_method: rebase
+      merge: {}
```

Comment:
do we need to keep merge action here? if we remove this won't the above default be applied?

Reply:
yes, I think it is needed, otherwise the action will not get executed